src/Felix/Instances/Function/Lift.agda

```diff
@@ -8,10 +8,24 @@ open Lift
 open import Data.Product using (_,_)
 
 open import Felix.Homomorphism hiding (refl)
+open import Felix.Object
 
 private module F {ℓ} where open import Felix.Instances.Function ℓ public
 open F
 
+private
+  variable
+    A : Set a
+
+lift₀ : ⦃ _ : Products (Set (a ⊔ b)) ⦄ → A → (⊤ F.⇾ Lift b A)
```

Comment:
Considering the "`open F`" above, do you need the "`F.`" here and below?

Reply:
I've missed this comment somehow.

I'm not sure why I did this in the first place, probably copied it from outside felix like so.